Constants.mk

```diff
@@ -0,0 +1,78 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
```

Comment:
Correct! My bad, I will rename it.

How about `Commonvars.mk` or `EnvVars.mk`?